src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Connection.cs

```diff
@@ -86,6 +86,7 @@ internal Connection()
         public void Start()
         {
             Log.ConnectionStart(ConnectionId);
+            KestrelEventSource.Log.ConnectionStart(this);
```

Comment:
Event source is conceptually different from logging. It didn't seem appropriate to be event sourcing inside the logger implementation.